viewflow_extensions/management/commands/create_diagraph.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import absolute_import, unicode_literals
+
+import importlib
+import os
+from collections import defaultdict
+
+from django.conf import settings
+from django.core.management import BaseCommand
+from viewflow import flow
+
+from viewflow_extensions.utils import GraphvizException, is_flow_task
+
+
+class Command(BaseCommand):
+    help = 'Creates diagraph for the given flow.'
+    args = '<app> <flow>'
+
+    @staticmethod
+    def get_next(node):
+        if isinstance(node, flow.If):
+            return [getattr(node, '_on_true'), getattr(node, '_on_false')]
+        if isinstance(node, flow.Switch) or isinstance(node, flow.Split):
+            branches = getattr(node, 'branches')
+            return [b[0] for b in branches]
+        if not isinstance(node, flow.End):
+            return getattr(node, '_next')
+
+    def create_graph(self, flow_cls):
+        graph = defaultdict(list)
+        node_list = [
+            getattr(flow_cls, item)
+            for item in dir(flow_cls)
+            if is_flow_task(getattr(flow_cls, item))
+        ]
+        for node in node_list:
+            next_nodes = Command.get_next(node)
+            if next_nodes:
+                if type(next_nodes) == list:
+                    graph[node] += next_nodes
+                else:
+                    graph[node].append(next_nodes)
+        return graph
+
+    def get_diagraph(self, flow_cls):
+        diagraph = """
+         digraph {
+            graph [rankdir=TB]
+            node [shape=box, style=rounded, labelfloat=true]
+            edge [labelfloat=true]
+        """
+        diagraph += "graph [label=" + str(flow_cls) + "] {"
+        graph = self.create_graph(flow_cls)
+        nodes = ""
+        connections = ""
+        for key, val in graph.items():
+            nodes += "%s \n" % (key.name,)
+
+            for i in val:
+                connections += "%s -> %s \n" % (key.name, i.name)
+
+        diagraph += nodes + connections + "}}"
+        return diagraph
+
+    def handle(self, *args, **options):
+        if not settings.DEBUG:
+            return
```

Comment:
why would i overwrite settings here? since it's only for debugging, i only want to allow using it in debug mode.